DOC-2363: Custom block elements with colon characters would throw errors. (#3259)
tinymce/tinymce-docs: modules/ROOT/pages/7.1-release-notes.adoc

{productname} 7.1 also includes the following bug fix<es>:

=== Custom block elements with colon characters would throw errors.
// #TINY-10813

> Previously, when using `custom_elements` containing characters like colons, and that element has an anchor element nested within an error would be thrown in the dev-console as the corresponding CSS selector was invalid due to the unescaped colon character.

Reviewer: ```suggestion
Previously, when `custom_elements` contained special characters such as colons, and these elements included a nested anchor element, an error would be thrown in the dev-console as the corresponding CSS selector was invalid due to the unescaped colon character.
```
